com.microsoft.mrtk.uxcore/Editor/Inspectors/FontIconSet/FontIconSelectorInspector.cs

```diff
@@ -28,15 +25,26 @@ class FontIconSelectorInspector : UnityEditor.Editor
         private SerializedProperty currentIconNameProp = null;
         private SerializedProperty tmProProp = null;
 
+        private GUIStyle currentButtonStyle;
+
+        private bool initializedStyle = false;
+
         private void OnEnable()
         {
             fontIconsProp = serializedObject.FindProperty("fontIcons");
             currentIconNameProp = serializedObject.FindProperty("currentIconName");
             tmProProp = serializedObject.FindProperty("textMeshProComponent");
+            
         }
 
         public override void OnInspectorGUI()
         {
+            if (!initializedStyle)
+            {
+                currentButtonStyle = new GUIStyle(GUI.skin.button);
+                initializedStyle = true;
+            }
```

Comment:
Could this be done in `OnEnable`? Or is there some edge case where we need to init like this?

Reply:
Yes; I believe there's edge cases like the prefab override window where `OnEnable` is unreliable. Folks online recommend this style. Not my idea! :D